anaUltraScurve.py

```diff
@@ -54,13 +54,12 @@
         pass
     pass
 
-buildHome = os.environ.get('BUILD_HOME')
-
+projectHome = os.environ.get('GEM_PLOTTING_PROJECT')
```

Comment:
Quick question, is there a check that is run before this point that this variable is set (and points somewhere real)?
If not, add an issue such that checks like this are made in the future